### ISSUE TITLE
Remove babel polyfill from ESM output

### ISSUE THREAD

**Issue by crimx** [react-component/field-form#119]
ESM is for modern usage and should not include babel runtime. The babel regenerator runtime requires `unsafe-eval` CSP which is a safety concern. Removing the babel polyfill can fix this issue.

**Comment by crimx** [comment]
> The babel regenerator runtime requires unsafe-eval CSP which is a safety concern.

Looks like they've [fixed](https://github.com/facebook/regenerator/pull/346) the issue. It was due to legacy dependencies.

Still recommend to remove the babel runtime for ESM though.

**Comment by guoyunhe** [comment]
Agree, we use polyfill.io and babel runtime is totally unnecessary in our scenario.
